deploy/stacks/pipeline.py

```diff
@@ -611,6 +610,38 @@ def set_db_migration_stage(
             ),
         )
 
+    def set_stacks_updater_stage(
```

Comment:
I still have doubts if we really need a dedicated stage to update stacks? 🤔 
1) `stack_updates` runs every night on schedule, so env stacks should automatically be updated on the next day (Is there a customer that needs to have all stack updates ASAP without waiting until next day?)
2) Okay, we can assume that this feature can also be used for some other changes in future. In this case, why wouldn't we introduce a button to update all stacks. We already have a button to update one stack, so it should be easy to do for many stacks. Otherwise, just to use this feature, a customer has to commit a change with `enable_update_dataall_stacks_in_cicd_pipeline=true`, deploy it, wait for downtime while all environment and dataset stacks are updated, and not to forget to commit `enable_update_dataall_stacks_in_cicd_pipeline=false` on the next round (or get more 1-2 hours of downtime). 

Please, feel free to correct me since I might have misunderstood something :)

Reply:
I think for prod deployments, the less downtime the better. In this case downtime is not just that the application does not work, users might try to perform actions that break the CloudFormation stacks that are not updated.
@dosiennik @kukushking I would also want your opinion.

Reply:
if I understood the question correctly, I think adding this stage is not only for the purpose of migration, like enabling it and disabling once the job of updating the stacks is done, the step is added to the pipeline also for the future changes into the pivot role and applying the changes as a part of the deployment - not to wait until night but to see the deployed changes/hot-fixes-  if something is missing in the pivot role - as soon as possible

Reply:
Couple of thoughts: for this specific pivot role change we do want to roll out the update as soon as possible to minimize downtime (and avoid "wait until midnight" type of instructions in the readme :)). Auto-update feature is one of the ways to achieve that (albeit with a bit of friction having to enable it) and it probably is a useful feature beyond this one change. 

I do like the idea of "update all" button though, as a way to force the update once. This will address the use case where users want to keep auto-update off (either for cost optimisation or something else). But all of this is probably out of scope of this PR, as long as there is an update path it's fine.

Reply:
Yep, sounds good!